src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -22,18 +22,18 @@ After you've completed setting up a project in Sentry, Sentry will give you a va
 
 You should `init` the Sentry Browser SDK as soon as possible during your page load:
 
-``` javascript
-Sentry.init({ dsn: 'https://<key>@sentry.io/<project>' });
-
+```javascript
 // When using npm, import Sentry
 import * as Sentry from '@sentry/browser';
+
+Sentry.init({ dsn: '___PUBLIC_DSN___' });
 ```
 
 ### Verifying Your Setup
 Great! Now that you've completed setting up the SDK, maybe you want to quickly test out how Sentry works. Start by capturing an exception:
 
 ```javascript
-Sentry.captureException(new Error("This is my fake error message"));
+Sentry.captureException(new Error("Something broke"));
```

Comment:
I've updated the screenshot and message since it was wrong.

Reply:
![error-message](https://user-images.githubusercontent.com/363802/59349437-ef4cd700-8d19-11e9-933d-06bc622e6a10.png)